content/docs/dvclive/api-reference/live/make_dvcyaml.md

```diff
@@ -0,0 +1,42 @@
+# Live.make_dvcyaml()
+
+Writes [DVC configuration](/doc/user-guide/project-structure/dvcyaml-files) for
+metrics, plots, and parameters to
+[`Live.dvc_file`](/doc/dvclive/api-reference/live#properties).
+
+```py
+def make_dvcyaml()
+```
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
+live.log_param("lr", 0.01)
+live.log_metric("acc", 0.9)
+live.log_sklearn_plot("confusion_matrix", [0, 0, 1, 1], [1, 0, 0, 1])
+live.make_dvcyaml()
+```
+
+## Description
+
+DVCLive will collect all the data logged in `{Live.dir}` and save the DVC
```

Comment:
Can we mention why this is needed?

Also, `collect all the data` - I think we can skip it. 


"Creates `dvc.yaml` file that describes plots, metrics, ... that were created with DVCLive. It's called automatically when ... . It's needed for tools like `dvc exp show` ... to show reports and experiment tables "

wdyt?